README.md

```diff
@@ -789,6 +762,8 @@ Everything else takes place inside the component or without impacting the previo
     display: block;
     float: right;
     margin: 0;
+    outline: 1px solid #ff0000;
```

Comment:
@teamjordy make these examples fully match our guidelines... change hex to `#f00`